### ISSUE TITLE
Add to README a tip about images when using command line

### ISSUE THREAD

**Issue by willingc** [freeman-lab/minidocs#66]
When using the command line to build a site (https://github.com/freeman-lab/minidocs#using-minidocs-on-the-command-line), the user must make sure that any of the images used in the markdown files are copied to the site directory. If the images can not be found when running a webserver and accessing the files in the site build directory, the image will not render.

A simple fix would be to edit the `README.md` to remind users to take care to copy the directory containing their images to the site build directroy.

**Comment by sethvincent** [comment]
Thanks for pointing this out!

We just added a nifty option to help with this, too.

the `--assets` option will copy a directory of images or other files into the built site directory.

example:

```sh
minidocs docs/ -c contents.json -o site/ --assets images/
```

You'll then have a `site/images` directory and can reference the files in the markdown files as `/images/whatever.jpg`

I added this example to the readme just now.
